examples/nidaqmx/analog-input-every-n-samples.py

```diff
@@ -64,6 +64,11 @@ def check_for_warning(response):
                     f"{warning_message.error_string}\nWarning status: {response.status}\n"
                 )
 
+        def check_for_stream_error(stream: grpc.Future) -> None:
```

Comment:
The gRPC Python documentation says the stream object is a `Future`, and I can call `Future` methods at run time, but grpc-stubs doesn't agree:
```
examples/nidaqmx/analog-input-every-n-samples.py:70: error: Exception must be derived from BaseException  [misc]
examples/nidaqmx/analog-input-every-n-samples.py:110: error: Argument 1 to "check_for_stream_error" has incompatible type "CallIterator[RegisterEveryNSamplesEventResponse]"; expected "Future[Any]"  [arg-type]
examples/nidaqmx/analog-input-every-n-samples.py:117: error: Argument 1 to "check_for_stream_error" has incompatible type "CallIterator[RegisterDoneEventResponse]"; expected "Future[Any]"  [arg-type]
Found 3 errors in 1 file (checked 7 source files)
```

I filed a grpc-stubs bug: https://github.com/shabbyrobe/grpc-stubs/issues/41

I guess I'll use `Any` for now.